File: Facade-Testing/src/Tuner.java
public class Tuner {
    Amplifier amp;
    boolean on;
    String freq;

    public void on(){
        this.on = true;
    }

    public void off(){
        this.on = false;
    }

    public void setAM(){
        freq = "AM";
        System.out.println("You are on AM frequency");
    }

    public void setFM(){
        freq = "FM";
        System.out.println("You are on FM frequency");
    }

    public void setFrequency(int frequency){
        System.out.println("You are on " + frequency + " " + freq);
    }
}

File: Facade Pattern C# version/FacadeConsole/Tuner.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace FacadeConsole
{
    public class Tuner
    {
        Amplifier amp;
        bool active = false;
        string freq;

        public Tuner() 
        {
            freq = "AM";
        }

        public void TurnOn()
        {
            active = true;
        }

        public void TurnOff()
        {
            active = false;
        }

        public void SetAM()
        {
            freq = "AM";
            Console.WriteLine("You are on AM frequency");
        }

        public void SetFM()
        {
            freq = "FM";
            Console.WriteLine("You are on FM frequency");
        }

        public void SetFrequency(int frequency)
        {
            Console.WriteLine("You are on " + frequency + " " + freq);
        }

        public void PlayMusic()
        {
            Console.WriteLine("You hear cool music from the radio");
        }
    }
}
